RUFAS/routines/manure/manure_treatments/manure_treatment_daily_output.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+
+from dataclasses import astuple
+from dataclasses import dataclass
+
+from RUFAS.routines.manure.protocols.liquid_manure_portion_protocol import LiquidManurePortionProtocol
+
+
+@dataclass
+class ManureTreatmentDailyOutput(LiquidManurePortionProtocol):
+    """Daily output of a manure treatment.
+
+    Attributes:
+        pen_id: ID of the pen that this output is associated with.
+        simulation_day: Number of days into the simulation.
+        liquid_manure_total_ammoniacal_nitrogen: Total ammoniacal nitrogen, kg.
+        liquid_manure_nitrogen: Amount of nitrogen in manure, kg.
+        liquid_manure_total_solids: Total amount of solids from the manure and the bedding, kg.
+        liquid_manure_total_volatile_solids: Total amount of volatile solids, kg.
+        liquid_manure_phosphorus: Amount of phosphorus excreted in manure, kg.
+        liquid_manure_potassium: Amount of potassium in manure, kg.
+        final_manure_volume: Final manure volume after treatment, m^3.
+
```

Comment:
I think you're missing a few attribute docstrings. is there a reason you excluded them? Perhaps they're documented in the parent class (it doesn't look like it, though).

```suggestion
        liquid_manure_potassium: Amount of potassium in manure, kg.
        final_manure_volume: Final manure volume after treatment, m^3.
        liquid_manure_daily_volume: [description of variable]
        storage_methane: [description]
        ...
        accumulated_final_manure_volume: [description]
```